### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by ghost** [googleapis/google-cloud-cpp#5595] [General]
This is an... interesting new feature?

Anyway introduction. I'm remyabel (remy abel is *not* my name, contrary to popular belief, although I found out that it looks like one). I used to be active on the C++ tag of Stackoverflow but left because it was too toxic. I was [contacted by google](https://meta.stackexchange.com/questions/214729/hello-from-google-spam-or-no) 6 years ago when I was still in high school. Now I voluntarily work on this project because I am acclimated to it (although I do try to contribute to other projects, I am so used to this one now that I don't branch out as much).

That'll be all, thanks everyone for letting me contribute!

**Comment by coryan** [comment]
Hi @remyabel, we should be thanking you for your contributions, the project is better because of your participation.  Good time for me to send a shout to all the other contributors from the open source community, whether you fixed a typo, or changed the GCS emulator (hi @vnvo2409 !), we are lucky to have your help.

**Comment by vnghia** [comment]
This feature is awesome. Let me introduce myself real quick !

I am Vo Van Nghia ( so vn + vo = vnvo ). I've just involved in open-source since this summer through [GSoC](https://summerofcode.withgoogle.com/archive/2020/projects/5669113902399488/). My work required some new features of `google-cloud-cpp` so eventually I became a contributor of this library. Afterwards, I found that this community is super friendly and I decided to become a regular contributor of it.

I learned a lot of thing thanks to everyone in this community. And shout out to @coryan, without you, I won't go this far !

**Comment by devjgm** [comment]
We're super happy to have contributors like @remyabel and @vnvo2409. Thanks!
